XTS/raw-xts-api-demo/test/pom.xml

```diff
@@ -118,7 +124,7 @@
                     <!-- Forked process timeout -->
                     <forkedProcessTimeoutInSeconds>${surefire.forked.process.timeout}</forkedProcessTimeoutInSeconds>
                     <!-- System properties to forked surefire JVM which runs clients. -->
-                    <argLine>${jvm.args.ip.client}</argLine>
+                    <argLine>${jvm.args.ip.client} ${modular.jdk.args}</argLine>
```

Comment:
is `${modular.jdk.args}` still needed?

Reply:
Thanks @jmfinelli , I can remove it now.